docs/migrating-v1beta1-to-v1.md

```diff
@@ -0,0 +1,63 @@
+<!--
+---
+linkTitle: "Migrating from Tekton v1beta1"
+weight: 4000
+---
+-->
+
+# Migrating From Tekton `v1beta1` to Tekton `v1`
+
+- [Changes to fields](#changes-to-fields)
+- [Upgrading `PipelineRun.Timeout` to `PipelineRun.Timeouts`](#upgrading-pipelinerun.timeout-to-pipelinerun.timeouts)
+- [Deprecating Resources from Task, TaskRun, Pipeline and PipelineRun](#deprecating-resources-from-task,-taskrun,-pipeline-and-pipelinerun)
+- [Replacing `taskRef.bundle` and `pipelineRef.bundle` with Bundle Resolver](#replacing-taskRef.bundle-and-pipelineRef.bundle-with-bundle-resolver)
+
+
+This document describes the differences between `v1beta1` Tekton entities and their
+`v1` counterparts. It also describes the changed fields and the deprecated fields into v1.
+## Changes to fields
+
+In Tekton `v1`, the following fields have been changed:
+
+| Old field | Replacement |
+| --------- | ----------|
+| `PipelineRun.Timeout`| `timeouts.pipeline` |
+| `spec.taskRunSpecs.taskServiceAccountName` | `spec.taskRunSpecs.serviceAccountName` |
+| `spec.taskRunSpecs.taskPodTemplate` | `spec.taskRunSpecs.podTemplate` |
+| `status.taskResults` | `status.results` |
+| `status.pipelineResults` | `status.results` |
+| `taskRun.spec.taskRef.bundle` | `taskRun.spec.taskRef.resolver` |
+| `pipelineRun.spec.pipelineRef.bundle` | `pipelineRun.spec.pipelineRef.resolver` |
+| `task.spec.resources` | removed from `Task` |
+| `taskrun.spec.resources` | removed from `TaskRun` |
+| `pipeline.spec.resources` | removed from `Pipeline` |
+| `pipelineRun.spec.resources` | removed from `PipelineRun` |
+
+## Deprecating `resources` from Task, TaskRun, Pipeline and PipelineRun
+Due to the deprecation of `PipelineResources`, the `resources` fields of Task, TaskRun, Pipeline and PipelineRun can be replaced by `Task` following the [reference](https://github.com/tektoncd/pipeline/blob/main/docs/pipelineresources.md)
+
+## Replacing `taskRef.bundle` and `pipelineRef.bundle` with Bundle Resolver
+Bundle resolver in remote resolution should be used instead of `taskRun.spec.taskRef.bundle` and `pipelineRun.spec.pipelineRef.bundle`
+
+```yaml
+# TaskRun.yaml (v1beta1)
+spec:
+  taskRef:
+    name: example-task
+    bundle: docker.io/ptasci67/example-oci@sha256:03
+
+# TaskRun.yaml (v1)
+spec:
+  taskRef:
+    resolver: bundles
+    params:
+    - name: bundle
+      value: foo.bar/baz
```

Comment:
can you please use a more realistic example here and above, and make sure they match?